packages/react-components/react-charts-preview/src/utilities/SVGTooltipText.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { ITooltipHost, ITooltipProps, Tooltip, TooltipDelay } from '@fluentui/react/lib/Tooltip';
-import { Async, KeyCodes, getId, portalContainsElement } from '../Utilities';
+import { Async, KeyCodes, getId, portalContainsElement } from '@fluentui/react/lib/Utilities';
```

Comment:
This needs to be removed. Could you comment out the code using all v8 dependencies

Reply:
commented the svgTooltip component as discussed